USA/CA/alameda/README.md

```diff
@@ -1,6 +1,8 @@
 This readme should give people everything they need to maintain the scraper.
 
 # Summary
+This scraper saves data into a seperate folder if it has `daily` in the `name`. **DO NOT** attempt to update to use `list_pdf_scrapers` or `bs_common`.
```

Comment:
Can you document why we shouldn't attempt this?

Reply:
Fixed in 3de2238